content/en/users/online-storage/block-storage/_index.md

```diff
@@ -0,0 +1,662 @@
+|                              |   'attachment_id': '291111d3-f43c-494c-b64c-5c0abcda3d37', |
+|                              |   'attached_at': '2021-08-05T08:50:19.000000',             |
+|                              |   'host_name': 'sbgcsrv17.in2p3.fr',                       |
+|                              |   'volume_id': '9a4000fb-0bcc-47e8-96fb-85a222295402',     |
+|                              |   'device': '/dev/vdb',                                    |
+|                              |   'id': '9a4000fb-0bcc-47e8-96fb-85a222295402'}]           |
+| availability_zone            | nova                                                       |
+| bootable                     | false                                                      |
+| consistencygroup_id          | None                                                       |
+| created_at                   | 2021-08-05T08:48:41.000000                                 |
+| description                  |                                                            |
+| encrypted                    | False                                                      |
+| id                           | 9a4000fb-0bcc-47e8-96fb-85a222295402                       |
+| multiattach                  | False                                                      |
+| name                         | Matlab                                                     |
+| os-vol-tenant-attr:tenant_id | 7a910xxxxae74ed9yyyy7497zzzz9499                           |
+| properties                   |                                                            |
+| replication_status           | None                                                       |
+| size                         | 50                                                         |
+| snapshot_id                  | None                                                       |
+| source_volid                 | None                                                       |
+| status                       | in-use                                                     |
+| type                         | ceph                                                       |
+| updated_at                   | 2021-08-05T08:50:20.000000                                 |
+| user_id                      | babzz8c3b4cxxxx6286dacyyyy01e5a3                           |
++------------------------------+------------------------------------------------------------+
+```
+
+### Attach volume to VM
+
+Mapping block devices to VMs is described in detail in the
+[OpenStack documentation](https://docs.openstack.org/nova/wallaby/user/block-device-mapping.html).
+
+To
+[attach a volume to a VM](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/server.html#server-add-volume)
+use the FedCloud command below:
+
+{{% alert title="Note" color="info" %}} To be able to attach a volume to a VM,
```

Comment:
there is no need to stop a VM to attach a volume, this can be done on the fly

Reply:
The commands I tried were failing if the VM was running.